webpack.config.dev.js

```diff
@@ -21,7 +21,15 @@ module.exports = {
   },
   module: {
     loaders: [
-      { test: /\.jsx?$/, loaders: ['react-hot', 'babel-loader?experimental'], exclude: /node_modules/ }
+      {
+        test: /\.jsx?$/,
+        loaders: ['react-hot', 'babel-loader?experimental'],
+        exclude: /node_modules/
+      },
+      {
+        test: /\.scss$/,
```

Comment:
I had to add this to the webpack config to tell it to use the loade for `.scss` files.